card/mod/core/chunk/reference.rb

```diff
@@ -8,12 +8,15 @@ class Reference < Abstract
 
         def referee_name
           return if name.nil?
-          @referee_name ||= begin
-            rendered_name = render_obj(name)
-            ref_card = fetch_referee_card rendered_name
-            ref_card ? ref_card.name : rendered_name.to_name
-          end
+          @referee_name ||= referee_name_from_rendered(render_obj(name))
           @referee_name = @referee_name.absolute(card.name).to_name
+        rescue Card::Error::NotFound
```

Comment:
fixes voting bug.  this continues the (debatable) strategy of allowing pointers to contain illegal references, or at least not breaking weirdly when they do. Our current situation on wikirate is that many pointers have id references to cards that have since been deleted. Arguably, if we're using ids, we have to enforce referential integrity (ie, you can't delete a card without deleting all references to its id).  names do not have this problem...

anyway, I don't want to get stuck on voting stuff when we're about to replace it with bookmarks, so this is a quick fix.